scripts/download_booster_rules.js

```diff
@@ -77,8 +77,12 @@ const getCard = (cardCode) => {
 
 const getUuid = (cardCode) => {
   const [setCode, cardNumber] = cardCode.split(":");
-  const { cardsByNumber } = getSet(setCode.toUpperCase());
```

Comment:
this line here was the gotcha - destructuring `undefined` was causing an error to be thrown in CI

I don't know why the set would not be found, maybe that's an error which should throw, as it shows your data is somehow really wrong.